app/client/components/ViewLayout.ts

```diff
@@ -294,7 +294,7 @@ export function buildViewSectionDom(options: {
     dom.cls('active_section', vs.hasFocus),
     dom.cls('active_section--no-indicator', !focusable),
     dom.maybe<BaseView|null>((use) => use(vs.viewInstance), (viewInstance) => dom('div.viewsection_title.flexhbox',
-      dom('span.viewsection_drag_indicator.glyphicon.glyphicon-option-vertical',
+      cssDragIcon('DragDrop',
```

Comment:
```suggestion
      cssDragIcon('DragDrop',
      dom.cls("viewsection_drag_indicator"),
```
Can you add this class, there are some internal tests looking for it.